src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj

```diff
@@ -28,11 +22,9 @@
     <ProjectReference Include="..\..\Core\Portable\Microsoft.CodeAnalysis.Workspaces.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
     <PackageReference Include="System.Composition" Version="$(SystemCompositionVersion)" />
+    <PackageReference Include="System.ComponentModel.Composition" Version="$(SystemComponentModelCompositionVersion)" />
```

Comment:
This was added because we're still compiling this against .NET Framework, and we need to pick up the two argument System.Lazy; somehow this was working because of a dependency of Microsoft.VisualStudio.CodingConventions but it wasn't obvious to me what that dependency was. Suggestions welcome here.